### PR TITLE
Convert project tags page to React

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/templates/sentry/projects/manage.html [hunk 1]
```diff
@@ -206,8 +206,8 @@ <h6 class="nav-header">{% trans "Configuration" %}</h6>
     <li{% if page == 'alerts' %} class="active"{% endif %}>
         <a href="{% absolute_uri '/{}/{}/settings/alerts/' project.organization.slug project.slug %}">{% trans "Alerts" %}</a>
     </li>
-    <li{% if page == 'tags' %} class="active"{% endif %}>
-        <a href="{% url 'sentry-manage-project-tags' project.organization.slug project.slug %}">{% trans "Tags" %}</a>
+    <li>
+        <a href="{% absolute_uri '/{}/{}/settings/tags' project.organization.slug project.slug %}">{% trans "Tags" %}</a>
```
Comment (billyvg) [inline src/sentry/templates/sentry/projects/manage.html]: I haven't tested this but I suspect URL is wrong here, missing trailing slash

File: src/sentry/static/sentry/app/views/projectTags.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                      <h5>
+                        {name}
+                        &nbsp;
+                        <small>({key})</small>
+                      </h5>
+                    </td>
+                    <td>
+                      <a
+                        className="btn btn-sm btn-default"
+                        onClick={() => this.onDelete(key, idx)}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectTags.jsx]: Looks like one thing we're missing here is the confirm modal before deleting. I believe we have a `<Confirm>` component for this.

**Review by billyvg — CHANGES_REQUESTED** [review]
Needs some small fixes, but otherwise looks great!

**Review by billyvg — APPROVED** [review]
Woo only 3 more views to go

**Comment by billyvg** [comment]
Not sure why org stats view is failing on percy, should be unrelated to this PR

**Pull request merged**